pandas/io/tests/test_gbq.py

```diff
@@ -56,6 +56,32 @@ def _skip_if_no_private_key_contents():
         _skip_if_no_private_key_contents()
 
 
+def _check_if_can_get_correct_default_credentials():
+    try:
+        from oauth2client.client import GoogleCredentials
+        from apiclient.discovery import build
+        credentials = GoogleCredentials.get_application_default()
+        bigquery_service = build('bigquery', 'v2', credentials=credentials)
+        jobs = bigquery_service.jobs()
+        job_data = {'configuration': {'query': {'query': 'SELECT 1'}}}
+        jobs.insert(projectId=PROJECT_ID, body=job_data).execute()
+        return True
+    except:
+        return False
+
+
+def _skip_if_cant_get_correct_default_credentials():
+    if not _check_if_can_get_correct_default_credentials():
+        raise nose.SkipTest("Cannot get default_credentials "
+                            "from the environment!")
+
+
+def _skip_if_can_get_correct_default_credentials():
```

Comment:
ok. pls just move them directly into the tests then as you are using them only once each (keep the ``_check_if_can......`` one though. that is DRY.